test/case/packaging/main.cpp

```diff
@@ -0,0 +1,6 @@
+#include <uxr/agent/Agent.hpp>
```

Comment:
Is it enough just with an include?
Shall que call some real code here?

Reply:
Real code is tested at `AgentUnitTest` but we can add some API function calls. What do you think?